### ISSUE TITLE
Benchmark `TFTNativeModel`

### ISSUE THREAD

**Issue by egoriyaa** [etna-team/etna#263]
### 🚀 Feature Request

Benchmark `TFTNativeModel` on `traffic` and `electricity` datasets

### Proposal

Conduct the same experiments as in [paper](https://arxiv.org/pdf/1912.09363.pdf). Log metrics in `wandb`

**Configs for `electricity`:**
Data: 
* 500k samples taken between 2014-01-01 to 2014-09-01
* iterative 24h forecasting for 7 days after train range

Features:
* Reals
  * `target`, `day-of-week`, `hour-of-day`, `time index` (the number of time steps from the first observation)
* Categoricals
  * `segment`

Transforms
* `StandartScalerTransform` for reals

Hyperparameters:
* n_segment: 370
* encoder_length: 168
* decoder_length: 24
* dropout: 0.1
* hidden_size: 160
* n_heads: 4
* batch_size: 64
* lr: 0.001
* max gradient norm: 0.1

**Configs for `traffic`:**
Data: 
* 500k samples taken before 2008-06-15
* iterative 24h forecasting for 7 days after train range

Features:
* The same as for `electricity`

Hyperparameters:
* n_segment: 440
* encoder_length: 168
* decoder_length: 24
* dropout: 0.3
* hidden_size: 320
* n_heads: 4
* batch_size: 128
* lr: 0.001
* max gradient norm: 100
